src/main/java/com/venafi/vcert/sdk/VCertTknClient.java

```diff
@@ -0,0 +1,247 @@
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CertificateRequest generateRequest(ZoneConfiguration config, CertificateRequest request, String accessToken)
+            throws VCertException {
+        try {
+            return connector.generateRequest(config, request, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    @Override
+    public String requestCertificate(CertificateRequest request, String zone, String accessToken) throws VCertException {
+        try {
+            return connector.requestCertificate(request, zone, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String requestCertificate(CertificateRequest request, ZoneConfiguration zoneConfiguration, String accessToken)
+            throws VCertException {
+        try {
+            return connector.requestCertificate(request, zoneConfiguration, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PEMCollection retrieveCertificate(CertificateRequest request, String accessToken) throws VCertException {
+        try {
+            return connector.retrieveCertificate(request, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void revokeCertificate(RevocationRequest request, String accessToken) throws VCertException {
+        try {
+            connector.revokeCertificate(request, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String renewCertificate(RenewalRequest request, String accessToken) throws VCertException {
+        try {
+            connector.renewCertificate(request, accessToken);
+        } catch (FeignException e) {
+            throw VCertException.fromFeignException(e);
+        } catch (Exception e) {
+            throw new VCertException("Unexpected exception", e);
+        }
+        return null;
```

Comment:
```suggestion
        try {
            return connector.renewCertificate(request, accessToken);
        } catch (FeignException e) {
            throw VCertException.fromFeignException(e);
        } catch (Exception e) {
            throw new VCertException("Unexpected exception", e);
        }
```

Reply:
previously we were returning always null in case of success

Reply:
Updated relevant methods to return the appropriate object instead of null.